src/InstrumentationEngine.Packages.sln

```diff
@@ -47,95 +47,121 @@ EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Debug|Any CPU = Debug|Any CPU
+		Debug|ARM64 = Debug|ARM64
 		Debug|x64 = Debug|x64
 		Debug|x86 = Debug|x86
 		Release|Any CPU = Release|Any CPU
+		Release|ARM64 = Release|ARM64
 		Release|x64 = Release|x64
 		Release|x86 = Release|x86
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
 		{0CA8A56A-B19D-4B69-82A2-3DCBD95F8A05}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{0CA8A56A-B19D-4B69-82A2-3DCBD95F8A05}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{0CA8A56A-B19D-4B69-82A2-3DCBD95F8A05}.Debug|ARM64.ActiveCfg = Debug|Any CPU
 		{0CA8A56A-B19D-4B69-82A2-3DCBD95F8A05}.Debug|x64.ActiveCfg = Debug|Any CPU
 		{0CA8A56A-B19D-4B69-82A2-3DCBD95F8A05}.Debug|x86.ActiveCfg = Debug|Any CPU
 		{0CA8A56A-B19D-4B69-82A2-3DCBD95F8A05}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{0CA8A56A-B19D-4B69-82A2-3DCBD95F8A05}.Release|Any CPU.Build.0 = Release|Any CPU
+		{0CA8A56A-B19D-4B69-82A2-3DCBD95F8A05}.Release|ARM64.ActiveCfg = Release|Any CPU
+		{0CA8A56A-B19D-4B69-82A2-3DCBD95F8A05}.Release|ARM64.Build.0 = Release|Any CPU
 		{0CA8A56A-B19D-4B69-82A2-3DCBD95F8A05}.Release|x64.ActiveCfg = Release|Any CPU
 		{0CA8A56A-B19D-4B69-82A2-3DCBD95F8A05}.Release|x86.ActiveCfg = Release|Any CPU
 		{3D3BA42C-4A1E-4135-B750-4D8CB0A7117F}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{3D3BA42C-4A1E-4135-B750-4D8CB0A7117F}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{3D3BA42C-4A1E-4135-B750-4D8CB0A7117F}.Debug|ARM64.ActiveCfg = Debug|arm64
+		{3D3BA42C-4A1E-4135-B750-4D8CB0A7117F}.Debug|ARM64.Build.0 = Debug|arm64
```

Comment:
>Build [](http://example.com/codeflow?start=53&length=5)

The headers originally build for AnyCPU and grab the x86/x64 files. 

I'm ok with this design which requires us to do an explicit ARM64 build to grab AnyCPU (x86/x64)+ ARM64 files, that way we can ship the public headers package without ARM64. 

In the future I think we should have the AnyCPU build grab everything once we officially support ARM64.

Reply:
In the ConfigurationManager you can switch the Debug/Release configuration setting and check/uncheck the required boxes